docs/content/documentation/systems/Materials/heat_conduction/SemiconductorLinearConductivity.md

```diff
@@ -0,0 +1,25 @@
+<!-- MOOSE Documentation Stub: Remove this when content is added. -->
+
+# SemiconductorLinearConductivity
+!syntax description /Materials/SemiconductorLinearConductivity
+
+For Linear relationship in Steinhart Hart equation, coefficient C = 0 \\
```

Comment:
Introduce and spell out the whole Steinhart-Hart equation with the C term before saying that this model is for the case when C=0.